src/libYARP_OS/CMakeLists.txt

```diff
@@ -188,7 +188,6 @@ set(YARP_OS_IMPL_HDRS include/yarp/os/impl/AuthHMAC.h
                       include/yarp/os/impl/PortCorePacket.h
                       include/yarp/os/impl/PortCorePackets.h
                       include/yarp/os/impl/PortCoreUnit.h
-                      include/yarp/os/impl/PortManager.h
```

Comment:
I guess all the public headers contained in the `impl` directories are not actually part of the public API, but it would be nice to be explicit and document this explicitly in https://github.com/robotology/yarp/blob/master/.github/CONTRIBUTING.md#deprecated-features .

Reply:
:+1: I will add a comment there.
I also noticed that the supported systems are outdated.